src/goto-programs/adjust_float_expressions.cpp

```diff
@@ -81,8 +81,8 @@ void adjust_float_expressions(exprt &expr, const exprt &rounding_mode)
     return;
 
   // recursive call
-  for(auto &op : expr.operands())
-    adjust_float_expressions(op, rounding_mode);
+  Forall_operands(op, expr)
```

Comment:
What's the reason for changing back to the old macros?

Reply:
Just consistency with the rest of the file - are we incrementally moving to range-based for loops now then? If so, I'll drop this hunk.